tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/EtcTypeChecker.scala

```diff
@@ -363,7 +375,7 @@ class EtcTypeChecker(varPool: TypeVarPool, inferPolytypes: Boolean = true) exten
     listener.onTypeError(sourceRef, message)
   }
 
-  // tired of reading "arguments(s)", it's easy to fix
+  // tired of reading "argument(s)", it's easy to fix
```

Comment:
While we're at it, shall we make this comment more meaningful? :)

```suggestion
  // Pluralize the string "argument"
```